src/app/app-routing.module.ts

```diff
@@ -3,7 +3,7 @@ import { NgModule } from '@angular/core';
 import { MainCoreLegacyComponent } from '@components/main-core-legacy/main-core-legacy-component';
 import { MainComponentLegacyComponent } from '@components/main-legacy/main-legacy.component';
```

Comment:
The floating label stop working after you focus in the field.
![Screen Recording 2019-03-19 at 9 01 46 AM](https://user-images.githubusercontent.com/751424/54616855-f3709d80-4a25-11e9-836a-3e49ee4f9e7c.gif)

Reply:
done